.github/workflows/code_check.yml

```diff
@@ -9,6 +9,11 @@ on:
       - synchronize
       - reopened
 
+permissions:
```

Comment:
Interesante, no conocia esta propieda de permisos a nivel del workflow, aprendí algo nuevo.

Reply:
Lo vi aqui: https://docs.github.com/en/actions/security-for-github-actions/security-guides/automatic-token-authentication#permissions-for-the-github_token